frontend/src/components/UploadArea.tsx

```diff
@@ -36,25 +42,54 @@ const UploadArea = ({projectName, uploadVerification} : IUploadAreaProps) => {
                     <Box textAlign='center' sx={{ mt: 2 }}>
                         <Button variant="contained" sx={{ mr: 1 }} onClick={doVideoUpload}>Upload File</Button>
                         <Button variant="contained" sx={{ mr: 1 }} color="warning" onClick={removeFile}>Remove File</Button>
-                        <Button disabled={!isVideoUploaded} variant="contained" color="warning" onClick={frameGeneration}>Generate Frames</Button>
                     </Box>
-                    {loading && 
-                        <LinearProgress sx={{marginTop: 5}}></LinearProgress>
+                    <>
+                        {
+                            isVideoUploaded &&
+                            <video width="80%" style={{
```

Comment:
Das funktioniert nicht in Firefox.
![image](https://github.com/user-attachments/assets/1873533b-d4fb-43af-b9aa-4614cf89eff2)

Reply:
Das klappt jetzt auch in Firefox.